ldk/javascript/src/whisper/types.ts

```diff
@@ -238,13 +240,35 @@ export type Message = WhisperComponent<WhisperComponentType.Message> & {
 };
 
 export type DropZone = WhisperComponent<WhisperComponentType.DropZone> & {
+  /**
+   * Specify what extensions the user can include. Do not include the dot.
+   *
+   * @example ['jpg','jpeg','png']
+   */
   accept?: string[];
   label: string;
+  /**
+   * The number of files that can be selected. There is a hard limit of ten files that can be selected at once.
+   */
   limit?: number;
+  /**
+   * If provided, replaces the word "files" in the component.
+   */
   noun?: string;
+  /**
+   * The callback function to call whenever the user selects or unselects a file.
+   */
   onDrop: WhisperHandlerWithParam<File[]>;
   tooltip?: string;
   validationError?: string;
+  /**
+   * The value property can be used to update the list of files selected, including dropping them. If the value property
+   * is null or undefined, the existing selection will remain as-is. To deselect all the files, set it to an empty
+   * array.
+   *
+   * You cannot add a file to the selection via this interface. The component will ignore any files provided it
+   * currently does not have selected.
```

Comment:
"currently does not have selected." Is this WIP? Confused by that last bit.

Reply:
It's intended to be final; If the Loop author provides a File object they had stored from a previous event but that file has since been deselected, that file object will be ignored.

Reply:
Got it.

Reply:
_The component will ignore any provided files that are not currently selected_